### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by arzt**
using ubuntu vivid
using package openjdk-8-jdk (available from ubuntu vivid on)
removed python support for PredictionIO (not needed anymore)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,10 +8,9 @@ ENV HBASE_VERSION 1.0.0
 
 ENV PIO_HOME /PredictionIO-${PIO_VERSION}
 ENV PATH=${PIO_HOME}/bin:$PATH
-ENV JAVA_HOME /usr/lib/jvm/java-7-openjdk-amd64
+ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
 
-RUN apt-get update && apt-get install -y curl openjdk-7-jdk libgfortran3 python-pip
-RUN pip install predictionio
+RUN apt-get update && apt-get install -y curl openjdk-8-jdk libgfortran3
```
Comment (fabiankrack) [inline Dockerfile:13]: Why did you remove the python-pip package?

Comment (arzt) [inline Dockerfile:13]: it was just added to install the predictionio python sdk, which it's not needed for prediction io to run, when using java/scala only.

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM ubuntu
+FROM ubuntu:vivid
```
Comment (hajoeichler) [inline Dockerfile:1]: We should use LTS versions, or am I wrong for docker containers @onibox ?

**Comment by smalot** [comment]
Currently, the official ubuntu release is Xenial, so JDK packages hasn't got the same release numbers ...
